slider.css

```diff
@@ -0,0 +1,64 @@
+#carousel {
+    margin: auto;
+    padding-top: 50px;
+    max-width: 940px;
+	text-align: center;
+	font-size: 32px;
+    position: relative;
+    background-color: rgba(240,108,100,1);
+}
+.item {
+    margin: 0 auto;
+    width: 100%;
+	overflow: hidden;
+    height: 600px;
+	display: flex;
+	justify-content: center;
+    align-items: center;
+    background-color: rgba(240,108,100,1);
+}
+.item-1{
+    background: url(/singolo/images/slide-1.png) no-repeat;
+    background-size: contain;
+    background-position: center;
+}
+.item-2{
+    background: url(/singolo/images/slide-2.png) no-repeat;
+    background-size: contain;
+    background-position: center;
+    background-color: rgba(100,139,240,1);
+}
+.arrow-right {
+    width: 0; 
+    height: 0; 
+    border-top: 15px solid transparent;
+    border-bottom: 15px solid transparent;
+    border-left: 15px solid rgba(233,67,72,1);
+    position: absolute;
+    right: 0%;;
+    top: 40%;
+}
+  
+  .arrow-left {
+    width: 0; 
+    height: 0; 
+    border-top: 16px solid transparent;
+    border-bottom: 16px solid transparent; 
+    border-right:16px solid  rgba(233,67,72,1); 
+    position: absolute;
+    left: 0%;
+    top: 40%;
+}
+.img-background {
```

Comment:
#carousel,
.item,
.img-background {
background-color: rgba(240,108,100,1);
}
Or it would be even better (again :) ) to create a reusable class for this backgroud color.

Reply:
fixed